packages/nc-gui/components/smartsheet/expanded-form/index.vue

```diff
@@ -811,7 +818,7 @@ export default {
 }
 
 .nc-expanded-cell-header {
-  @apply w-full text-gray-700 xs:mb-2;
+  @apply w-full !sm:text-gray-500 !xs:text-gray-600 xs:mb-2;
```

Comment:
`!` is actually not required as long as one didn't add `!text-gray-500` on `<LazySmartsheetHeaderCell>` / `<LazySmartsheetHeaderVirtualCell>`. Due to mobile first, `text-gray-500` is not even required on both tag since `sm:` one would cover it already.

Currently it is like 
- default: text-gray-500
- xs: text-gray-600 overriding the default one
- sm: text-gray-500 again overriding xs and default one (viewpoint take higher precedence)

![image](https://github.com/nocodb/nocodb/assets/35857179/1e689b8b-b4cb-4c81-841f-225feec740bc)

Reply:
@wingkwong Have resolved your suggestions